src/utils/Utils.java

```diff
@@ -50,11 +56,11 @@ public void handle(WindowEvent event) {
 	}
 
 	public static void createWindow(Stage stage, Object parent, String fxmlLocation, String sceneTitle, Object userData) {
-		createWindow(stage, parent, fxmlLocation, sceneTitle, userData, null, null, null);
+		createWindow(stage, parent, fxmlLocation, sceneTitle, userData, null, null, null, null);
 	}
 
-	public static void createWindow(Stage stage, Object parent, String fxmlLocation, String sceneTitle, Object userData, String cssLocation) {
-		createWindow(stage, parent, fxmlLocation, sceneTitle, userData, cssLocation, null, null);
+	public static void createWindow(Stage stage, Object parent, String fxmlLocation, String sceneTitle, Object userData, String cssLocation, ResourceBundle bundle) {
```

Comment:
Instead of modifying this method, lets create another one that receives the bundle String.